cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,55 @@
 /// <reference types='cypress' />
+import { generateWorker } from '../support/generate';
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should check pagination', () => {
+    cy.get('.-pageInfo').should('be.visible').and('exist');
+    cy.get('[aria-label="rows per page"]').should('be.visible').and('exist');
+  });
+
+  it('should add new user', () => {
+    const { email, firstName, lastName, age, salary, department } =
+   generateWorker();
+    cy.get('#addNewRecordButton').click();
+    cy.get('#firstName').type(firstName);
+    cy.get('#lastName').type(lastName);
+    cy.get('#userEmail').type(email);
+    cy.get('#age').type(age.toString());
+    cy.get('#salary').type(salary.toString());
+    cy.get('#department').type(department);
+    cy.get('#submit').click();
+    cy.get('.rt-table').should('contain', firstName);
+  });
+
+  it('should delete user', () => {
+    cy.get('#delete-record-2').click();
+    cy.get('.rt-table').should('not.contain', '#delete-record-2');
+  });
+
+  it.only('should delete all users', () => {
+    cy.get('[title="Delete"').click({ multiple: true });
+    cy.get('.rt-table').should('not.contain', '#delete-record-2');
+  });
```

Comment:
The test case 'should delete all users' is not comprehensive. After deleting all users, you should verify that the table is empty. You're only checking that the delete button for a specific user is not present, which is not the same as checking that all users have been deleted.